components/transfer/TransferConfirmModal.vue

```diff
@@ -0,0 +1,212 @@
+                    <span class="mx-2">
+                      <Identity
+                        :address="address.address"
+                        hide-identity-popover />
+                    </span>
+                    <NeoIcon
+                      icon="circle-info"
+                      pack="far"
+                      :title="address.address" />
+                  </div>
+                </div>
+                <div
+                  class="is-flex is-justify-content-space-between is-align-items-center">
+                  <span class="has-text-weight-bold">{{ $t('amount') }}</span>
+
+                  <div class="is-flex is-align-items-center">
+                    <span class="has-text-grey"
+                      >({{ address.token }} {{ unit }})</span
+                    >
+                    <span class="ml-1">${{ address.usd }}</span>
+                  </div>
+                </div>
+              </div>
+            </template>
+          </div>
+        </div>
+      </div>
+
+      <div class="is-flex is-flex-direction-column px-6 py-5 is-bordered-top">
+        <div
+          class="is-flex is-justify-content-space-between is-align-items-center mb-3">
+          <span class="has-text-weight-bold is-size-6">{{
+            $t('transfers.totalAmount')
+          }}</span>
+          <div class="is-flex is-align-items-center">
+            <span class="has-text-grey mr-1 is-small-size-text"
+              >({{ totalTokenAmount }} {{ unit }})</span
+            >
+
+            <span class="has-text-weight-bold is-size-5">
+              ${{ totalUsdValue }}</span
+            >
+          </div>
+        </div>
+
+        <NeoButton
+          :label="$t('teleport.send')"
+          variant="k-accent"
+          class="fixed-button-height is-flex is-flex-1"
+          @click.native="confirmTransfer" />
+      </div>
+    </div>
+  </NeoModal>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoIcon, NeoModal } from '@kodadot1/brick'
+import { NAMES } from '@/libs/static/src/names'
+import Avatar from '@/components/shared/Avatar.vue'
+import Identity from '@/components/identity/IdentityIndex.vue'
+import type { TargetAddress } from '@/components/transfer/Transfer.vue'
+const props = defineProps<{
+  isModalActive: boolean
+  totalTokenAmount: string
+  totalUsdValue: number
+  tokenIcon: string
+  unit: string
+  targetAddresses: TargetAddress[]
+}>()
+
+const isModalActive = useVModel(props, 'isModalActive')
+
+const { urlPrefix } = usePrefix()
+const emit = defineEmits(['close', 'confirm'])
+
+const closeModal = () => {
+  emit('close')
+}
+
+const confirmTransfer = () => {
+  closeModal()
+  emit('confirm')
+}
+const { accountId } = useAuth()
+
+const network = computed(() => NAMES[urlPrefix.value])
+const isExpandList = ref(false)
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.modal-width {
+  width: 28rem;
+}
```

Comment:
do we need to set a fixed width?

Reply:
Yes, it follows the design.

Reply:
aight was just waiting for some responsiveness